include/z64actor.h

```diff
@@ -288,8 +288,17 @@ typedef struct EnAObj {
     /* 0x000 */ Actor actor;
     /* 0x144 */ EnAObjActionFunc actionFunc;
     /* 0x148 */ ColliderCylinder collision;
-    /* 0x194 */ UNK_TYPE1 pad194[0x14];
-} EnAObj; // size = 0x1A8
+} EnAObj; // size = 0x194
+
+typedef enum {
+    /* 0 */ AOBJ_SIGNPOST_OBLONG,
+    /* 1 */ AOBJ_SIGNPOST_ARROW,
+} AObjType;
+
+#define AOBJ_GET_TEXTID(thisx) ((((thisx)->params >> 8) & 0xFF) | 0x300)
```

Comment:
Not sure how I feel about hiding the base textId like this, but we can worry about it later.